gulp-tasks/remark-lint-tests/check-html.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const url = require('url');
+const wfRegEx = require('../wfRegEx');
+const visit = require('unist-util-visit');
+const toString = require('mdast-util-to-string');
+
+module.exports = {
+  'wf-html-you-tube': wfYouTube,
+  'wf-html-dgc-links': wfHTMLDGCLinks,
+  'wf-html-internal-links': wfInternalLinks,
+  'wf-html-link-forced-lang': wfForcedLang,
+  'wf-html-unsafe-short-links': wfUnsafeShortLinks
+};
+
+const reYouTube = /^<iframe\s.*?src=['|"]https?:\/\/(www.)?youtube.com\/.*?['|"| ].*>[.\s\n\r]*?<\/iframe>$/i;
+const reDGCLink = /^<a\s.*?href=['|"]?(https?:)?\/\/developers.google.com/i;
+const reSandboxedLink = /^<a\s.*?href=['|"]?(https?:)?\/\/sandbox.google.com/i;
+const reGooGL = /^<a\s.*?href=['|"]?http:\/\/goo\.gl\//i;
+const reHref = /^<a\s.*href=['|"](.*)['|"]>/i;
+
+function wfYouTube(ast, file, setting) {
+  let msg = 'YouTube videos must use DevSite embed.';
+  visit(ast, 'html', function (node) {
+    if (reYouTube.test(node.value)) {
+      file.message(msg, node);
+    }
+  });  
+}
+
+function wfHTMLDGCLinks(ast, file, setting) {
+  let msg = 'Hard coded developers.google.com in link.';
```

Comment:
The other messages use phrases like 'must not' or 'do not'. You should probably be consistent.